pkg/abi/linux/fuse.go

```diff
@@ -643,6 +651,14 @@ func (r *FUSEDirents) MarshalUnsafe(dst []byte) {
 	}
 }
 
+// MarshalBytes serializes FUSEDirents to the dst buffer.
```

Comment:
I don't see this MarshalBytes used anywhere in the code, believe this also applies to MarshalUnsafe. Probably better to implement it when we actually use it?

Reply:
It is used in `NewRequest`. See other comment.